ui/DesignSystem/Component/Urn.svelte

```diff
@@ -1,25 +1,91 @@
 <script>
   import Copyable from "./Copyable.svelte";
-  import { Text } from "../Primitive";
+  import { Icon, Text } from "../Primitive";
 
   export let urn = null;
+  export let showOnHover = false;
+
+  let cleanUrn;
+  if (urn.includes("%rad:git:")) {
+    cleanUrn = urn.substring(9, urn.length);
+  } else if (urn.includes("rad:git:")) {
+    cleanUrn = urn.substring(8, urn.length);
+  } else {
+    cleanUrn = urn;
+  }
+
+  const firstSix = cleanUrn.substring(0, 7);
+  const lastSix = cleanUrn.substring(cleanUrn.length - 7, cleanUrn.length);
+
+  const fullUrn = { className: "hidden" };
```

Comment:
Why does this need to be an object?

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/731/commits/e4bf58400de29efcb6703c0cb0eb17f83ee78f7f